piptools/scripts/sync.py

```diff
@@ -41,6 +41,7 @@
     help="Add additional index URL to search",
     envvar="PIP_EXTRA_INDEX_URL",
 )  # noqa
+@click.option("--trusted-host", multiple=True, help="Add a custom trusted host")
```

Comment:
Use help string from the `pip` itself.

```suggestion
@click.option("--trusted-host", multiple=True, help="Mark this host as trusted, even though it does not have valid or any HTTPS.")
```